gdal/doc/source/drivers/raster/bsb.rst

```diff
@@ -6,6 +6,8 @@ BSB -- Maptech/NOAA BSB Nautical Chart Format
 
 .. shortname:: BSB
 
+.. built_in_by_default:: (can be disabled)
```

Comment:
```suggestion
.. built_in_by_default::
```
I suggest to remove the " (can be disable)". This is now true for all formats, at least with Unix builds